db/seeds.rb.example

```diff
@@ -0,0 +1,20 @@
+# Replace <your gmail name> with your gmail name below.
+
+Community.create!([
+  {name: "My Community", abbrv: "MC"}
+])
+
+Household.create!([
+  {unit_num: "0", community_id: 1, name: "Admins", deactivated_at: nil}
```

Comment:
`community = Community.create! ...`

`..., community: community, ...`

should work

The ID is different than the object. You can specify either.

On Fri, Sep 9, 2016 at 3:22 PM, Ken Winter <notifications@github.com> wrote:

> In db/seeds.rb.example
> <https://github.com/touchstone-cohousing/mess/pull/1#discussion_r78234869>
> :
>
> > @@ -0,0 +1,20 @@
> > +# Replace <your gmail name> with your gmail name below.
> > +
> > +Community.create!([
> > +  {name: "My Community", abbrv: "MC"}
> > +])
> > +
> > +Household.create!([
> > +  {unit_num: "0", community_id: 1, name: "Admins", deactivated_at: nil}
>
> What's the syntax for "saving the community into a variable"? The
> "community_id: Community.first" syntax exemplified in step 7 at
> https://github.com/touchstone-cohousing/mess doesn't work:
>
> ~/mess$ rake db:fake_user_data
> => QueryTrace disabled; CTRL-\ to toggle
> rake aborted!
> ActiveRecord::RecordInvalid: Validation failed: Community can't be blank
> /home/ken/mess/lib/tasks/fake_user_data.rake:11:in `block (2 levels) in '
> Tasks: TOP => db:fake_user_data
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/touchstone-cohousing/mess/pull/1/files/d49ab87ec9f05d1213b0a4b6bf614c19ebd67c5b#r78234869>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAq3cPahfW5h9c3NgEcrjFj2Si8_7Y8Yks5qobISgaJpZM4JzHfg>
> .
>



-- 
Tom Smyth

Worker-Owner, Sassafras Tech Collective
Specializing in innovative, usable tech for social change
sassafras.coop *·* @sassafrastech